src/NLU.DevOps.ModelPerformance/TestCaseSource.cs

```diff
@@ -226,6 +208,23 @@ bool isNoneIntent(string intent)
                     $"Expected intent '{expected}', actual intent '{actual}'.",
                     "Intent");
             }
+
+            foreach (string intent in intents)
+            {
+                if (intent != actual && intent != expected)
+                {
+                    yield return TrueNegative(
```

Comment:
Hmm, wondering if we really need to even bother tracking the true negatives, especially since we don't have the full model context.

Reply:
removed True negative calculations.